CONTRIBUTING.md

```diff
@@ -106,6 +106,23 @@ In order to work with that repository you will need:
 
 An IDE or text editor of your choice, like [Vim](https://www.vim.org/), [Emacs](https://www.gnu.org/software/emacs/) or [VS Code](https://code.visualstudio.com/) for example.
 
+### Add a `pre-push` git hook to check the lint before pushing
+
+By adding the following file `pre-push` (without extension) into your local git repository `.git/hooks` folder, the command `mvn spotless:chek` will run before every push you'll make to check if your code is still correctly linted, and will prevent pushing malformatted code which would fail your build anyway.
```

Comment:
```suggestion
By adding the following file `pre-push` (without extension) into your local git repository `.git/hooks` folder, 
the command `mvn spotless:chek` will run before every push you'll make to check if your code is still correctly linted, 
and will prevent pushing malformatted code which would fail your build anyway.
```